hack/jenkins/release_build_and_upload.sh

```diff
@@ -36,7 +36,7 @@ cat Makefile | grep "VERSION_MINOR ?=" | grep $VERSION_MINOR
 cat Makefile | grep "VERSION_BUILD ?=" | grep $VERSION_BUILD
 
 # Build and upload
-BUILD_IN_DOCKER=y make -j 16 all out/minikube-installer.exe out/minikube_${DEB_VERSION}.deb
+BUILD_IN_DOCKER=y make -j 16 all out/minikube-installer.exe out/minikube_${DEB_VERSION}.deb out/docker-machine-driver-hyperkit
```

Comment:
`make all` already includes both drivers

Reply:
I think I changed this right after the last release, so the drivers should be automatically uploaded the next time we run a release.

Reply:
Ah, ok. Makes sense. I didn't see it in there so I thought it was left out on purpose.

For the release, I thought I saw a different GCS location the kvm2 driver was published in:

gs://minikube/drivers/kvm/

Was that manual?